README.md

```diff
@@ -1,60 +1,40 @@
 # Project Name
 OOP-IN-RUBY
-> Description the project.
-
+> A project created just for learning Object-Oriented-Programming(OOP) in Ruby
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- RUBY
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+**instructions on setting up your project locally.**
 
```

Comment:
- [ ]  OPTIONAL
Kindly add a clear instructions in your ReadMe, on how to run your project. This will make your ReadMe more detailed